### ISSUE TITLE
Bug with online status when app lost focus

### ISSUE THREAD

**Issue by dary1337** [telegramdesktop/tdesktop#24834]
### Steps to reproduce

I have 2 Telegram accounts, with the help of the second account, I found out that the portable version of Telegram shows that My status is "online" when I lose focus, but this is not the case. If the application is in focus (for example, by clicking on Telegram in the taskbar), the online status disappears

### Expected behaviour

Online status is displayed incorrectly

### Actual behaviour

It seems that the online status is shown "the other way around"

### Operating system

Windows 10 Pro x64 (2009 build 19043)

### Version of Telegram Desktop

Portable (downloaded here: https://telegram.org/dl/desktop/win64_portable)

### Installation source

Static binary from official website

### Logs

```shell
[2022.07.28 17:35:53] Launched version: 4000002, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2022.07.28 17:35:53] Executable dir: Y:/PortableApps/Telegram/, name: Telegram.exe
[2022.07.28 17:35:53] Initial working dir: Y:/PortableApps/Telegram/
[2022.07.28 17:35:53] Working dir: Y:/PortableApps/Telegram/
[2022.07.28 17:35:53] Command line: Y:\PortableApps\Telegram\Telegram.exe
[2022.07.28 17:35:53] Executable path before check: Y:/PortableApps/Telegram/Telegram.exe
[2022.07.28 17:35:53] Logs started
[2022.07.28 17:35:53] Connecting local socket to Global\186e7783b9ac9ea0c8a614cc058cf4d9-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2022.07.28 17:35:53] This is the only instance of Telegram, starting server and app...
[2022.07.28 17:35:53] Moved logging from 'Y:/PortableApps/Telegram/log_start0.txt' to 'Y:/PortableApps/Telegram/log.txt'!
[2022.07.28 17:35:53] Primary screen DPI: 96
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.28 17:35:53] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.28 17:35:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.28 17:35:54] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.28 17:35:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.28 17:35:54] Using DirectX compiler 'Y:\PortableApps\Telegram\modules\x64\d3d\d3dcompiler_47.dll'.
[2022.07.28 17:35:54] Checking...
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\.
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\..
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\desktop.ini
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\File Explorer.lnk
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\Microsoft Edge.lnk
[2022.07.28 17:35:54] Checking C:\Users\yuri\AppData\Roaming\Microsoft\Internet Explorer\Quick Launch\User Pinned\TaskBar\Spotify.lnk
[2022.07.28 17:35:54] OpenAL Logging Level: (not set)
[2022.07.28 17:35:54] Audio Playback Devices: OpenAL Soft on yuri's Headphones (High Definition Audio Device);OpenAL Soft on Digital Audio (S/PDIF) (High Definition Audio Device)
[2022.07.28 17:35:54] Audio Playback Default Device: OpenAL Soft on yuri's Headphones (High Definition Audio Device)
[2022.07.28 17:35:54] Audio Capture Devices: OpenAL Soft on yuri's Microphone (High Definition Audio Device)
[2022.07.28 17:35:54] Audio Capture Default Device: OpenAL Soft on yuri's Microphone (High Definition Audio Device)
[2022.07.28 17:35:55] OpenGL: Using custom 'gpu_driver_bug_list.json'.
[2022.07.28 17:35:55] OpenGL Profile: OpenGLES.
[2022.07.28 17:35:55] OpenGL Renderer: ANGLE (NVIDIA, NVIDIA GeForce GTX 960 Direct3D11 vs_5_0 ps_5_0, D3D11-30.0.14.7141)
[2022.07.28 17:35:55] OpenGL Vendor: Google Inc. (NVIDIA)
[2022.07.28 17:35:55] OpenGL Version: OpenGL ES 3.0.0 (ANGLE 2.1.7 git hash: 0bb011f9e403)
[2022.07.28 17:35:55] OpenGL Extensions: GL_EXT_map_buffer_range, GL_EXT_texture_format_BGRA8888, GL_EXT_read_format_bgra, GL_EXT_robustness, GL_OVR_multiview2, GL_CHROMIUM_copy_compressed_texture, GL_EXT_disjoint_timer_query, GL_OES_packed_depth_stencil, GL_EXT_discard_framebuffer, GL_OES_texture_half_float, GL_ANGLE_framebuffer_blit, GL_NV_EGL_stream_consumer_external, GL_CHROMIUM_sync_query, GL_ANGLE_texture_compression_dxt3, GL_OES_texture_float_linear, GL_ANGLE_copy_texture_3d, GL_EXT_blend_func_extended, GL_ANGLE_program_cache_control, GL_OES_EGL_image_external_essl3, GL_OES_compressed_ETC2_RGB8_texture, GL_ANGLE_explicit_context, GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture, GL_OES_surfaceless_context, GL_OES_compressed_EAC_RG11_signed_texture, GL_ANGLE_pack_reverse_row_order, GL_EXT_debug_label, GL_EXT_sRGB, GL_EXT_EGL_image_external_wrap_modes, GL_ANGLE_lossy_etc_decode, GL_ANGLE_memory_size, GL_OES_draw_elements_base_vertex, GL_ANGLE_robust_client_memory, GL_OES_compressed_ETC2_sRGB8_alpha8_texture, GL_ANGLE_texture_usage, GL_EXT_draw_buffers, GL_EXT_unpack_subimage, GL_OES_depth32, GL_EXT_texture_norm16, GL_EXT_multisampled_render_to_texture, GL_OES_EGL_image_external, GL_CHROMIUM_copy_texture, GL_CHROMIUM_bind_uniform_location, GL_OES_compressed_EAC_R11_unsigned_texture, GL_EXT_draw_elements_base_vertex, GL_EXT_texture_compression_s3tc_srgb, GL_ANGLE_request_extension, GL_OES_compressed_EAC_RG11_unsigned_texture, GL_OES_depth24, GL_CHROMIUM_lose_context, GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture, GL_OES_compressed_EAC_R11_signed_texture, GL_EXT_float_blend, GL_OES_texture_border_clamp, GL_EXT_debug_marker, GL_EXT_texture_compression_bptc, GL_ANGLE_framebuffer_multisample, GL_NV_fence, GL_EXT_occlusion_query_boolean, GL_NV_pack_subimage, GL_OES_standard_derivatives, GL_OES_draw_buffers_indexed, GL_OES_fbo_render_mipmap, GL_ANGLE_instanced_arrays, GL_OES_texture_half_float_linear, GL_ANGLE_get_tex_level_parameter, GL_ANGLE_texture_multisample, GL_ANGLE_multiview_multisample, GL_ANGLE_provoking_vertex, GL_CHROMIUM_bind_generates_resource, GL_EXT_texture_compression_rgtc, GL_EXT_blend_minmax, GL_ANGLE_depth_texture, GL_ANGLE_explicit_context_gles1, GL_EXT_shader_texture_lod, GL_OES_get_program_binary, GL_EXT_color_buffer_float, GL_OVR_multiview, GL_OES_EGL_image, GL_OES_texture_npot, GL_OES_mapbuffer, GL_OES_texture_stencil8, GL_OES_vertex_array_object, GL_EXT_texture_filter_anisotropic, GL_KHR_parallel_shader_compile, GL_NV_pixel_buffer_object, GL_OES_texture_float, GL_OES_compressed_ETC2_sRGB8_texture, GL_ANGLE_base_vertex_base_instance, GL_ANGLE_get_serialized_context_string, GL_EXT_texture_compression_dxt1, , GL_ANGLE_texture_compression_dxt5, GL_EXT_color_buffer_half_float, GL_EXT_texture_storage, GL_OES_compressed_ETC2_RGBA8_texture, GL_ANGLE_multi_draw, GL_OES_element_index_uint, GL_EXT_texture_rg, GL_EXT_draw_buffers_indexed, GL_EXT_instanced_arrays, GL_KHR_debug, GL_WEBGL_video_texture, GL_ANGLE_translated_shader_source, GL_ANGLE_client_arrays, GL_EXT_frag_depth, GL_OES_rgb8_rgba8
[2022.07.28 17:35:55] EGL Extensions: EGL_EXT_create_context_robustness, EGL_ANGLE_d3d_share_handle_client_buffer, EGL_ANGLE_d3d_texture_client_buffer, EGL_ANGLE_surface_d3d_texture_2d_share_handle, EGL_ANGLE_query_surface_pointer, EGL_ANGLE_window_fixed_size, EGL_ANGLE_keyed_mutex, EGL_ANGLE_surface_orientation, EGL_ANGLE_direct_composition, EGL_ANGLE_windows_ui_composition, EGL_NV_post_sub_buffer, EGL_KHR_create_context, EGL_KHR_image, EGL_KHR_image_base, EGL_KHR_gl_texture_2D_image, EGL_KHR_gl_texture_cubemap_image, EGL_KHR_gl_renderbuffer_image, EGL_KHR_get_all_proc_addresses, EGL_KHR_stream, EGL_KHR_stream_consumer_gltexture, EGL_NV_stream_consumer_gltexture_yuv, EGL_ANGLE_flexible_surface_compatibility, EGL_ANGLE_stream_producer_d3d_texture, EGL_ANGLE_create_context_webgl_compatibility, EGL_CHROMIUM_create_context_bind_generates_resource, EGL_CHROMIUM_sync_control, EGL_EXT_pixel_format_float, EGL_KHR_surfaceless_context, EGL_ANGLE_display_texture_share_group, EGL_ANGLE_display_semaphore_share_group, EGL_ANGLE_create_context_client_arrays, EGL_ANGLE_program_cache_control, EGL_ANGLE_robust_resource_initialization, EGL_ANGLE_create_context_extensions_enabled, EGL_ANDROID_blob_cache, EGL_ANDROID_recordable, EGL_ANGLE_image_d3d11_texture, EGL_ANGLE_create_context_backwards_compatible, EGL_KHR_create_context_no_error, EGL_KHR_reusable_sync, 
[2022.07.28 17:35:55] OpenGL: QOpenGLContext created, version: 3.3.
[2022.07.28 17:35:55] OpenGL: [TRUE] (OverlayWidget)
[2022.07.28 17:35:55] Update Info: MTP is unavailable.
[2022.07.28 17:36:15] Export Info: Destroy top bar by controller removal.
[2022.07.28 17:36:15] API Warning: not loaded minimal channel applied.
[2022.07.28 17:36:15] API Warning: not loaded minimal channel applied.
[2022.07.28 17:36:16] App Info: writing encrypted user settings...
[2022.07.28 17:36:32] Export Info: Destroy top bar by controller removal.
[2022.07.28 17:36:33] App Info: writing encrypted user settings...
[2022.07.28 17:37:27] Error: could not locate 'background.jpg' in a zip file.
[2022.07.28 17:37:28] Error: could not locate 'background.jpg' in a zip file.
[2022.07.28 17:37:31] Error: could not locate 'background.jpg' in a zip file.
[2022.07.28 17:37:45] Export Info: Destroy top bar by controller removal.
[2022.07.28 17:37:57] RPC Error: request 228 got fail with code 0, error CLIENT_RESPONSE_PARSE_FAILED: Response parse failed.
[2022.07.28 17:37:57] App Fail: Failed to get featured stickers!
[2022.07.28 17:38:03] App Info: writing encrypted user settings...
[2022.07.28 17:38:27] Error: could not locate 'background.jpg' in a zip file.
[2022.07.28 17:38:51] Error: could not locate 'background.jpg' in a zip file.
```

**Comment by dary1337** [comment]
It's still actual, please fix

**Comment by iyear** [comment]
I encountered the same error.

**Comment by dary1337** [comment]
solved in 4.7.1 ver. Thx
